cli/switch_bin/switch.go

```diff
@@ -0,0 +1,147 @@
+package switch_bin
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/apex/log"
+	"github.com/fatih/color"
+	"github.com/manifoldco/promptui"
+	"github.com/tarantool/tt/cli/config"
+	"github.com/tarantool/tt/cli/list"
+	"github.com/tarantool/tt/cli/search"
+	"github.com/tarantool/tt/cli/util"
+	"github.com/tarantool/tt/cli/version"
+)
+
+// SwitchCtx contains information for switch command.
+type SwitchCtx struct {
+	// ProgramName is a program name to install.
+	ProgramName string
+	// Version of the program to install.
+	Version string
+}
+
+const ansi = "[\u001B\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*(?:;" +
+	"[a-zA-Z\\d]*)*)?\u0007)|(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PRZcf-ntqry=><~]))"
+
+func CleanString(str string) string {
```

Comment:
Public function requires a description. And why is it public?

Reply:
changed